packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/EverstakeFooter.tsx

```diff
@@ -33,9 +33,12 @@ export const EverstakeFooter = () => {
                 <EverstakeLogo color={isDarkMode ? '#fff' : '#000'} />
             </Left>
 
-            <TrezorLink href={HELP_CENTER_ETH_STAKING} target="_blank">
+            <ActionButton
```

Comment:
We use this logic 

```
            <StyledLink href={supportUrl} target="_blank">
                <Button variant="tertiary">
                    <Translation id="TR_EXCHANGE_DETAIL_CONVERTING_SUPPORT" />
                </Button>
            </StyledLink>
```

Reply:
Resolved in 19271b6f202293c7adc4fb1b10a1177dea997dd3

Reply:
Fixed some additional things in 89cfcfd7696429388e01e318a9ef5ab3699ebe6a